features/viewfolder/impl/src/main/kotlin/io/element/android/features/viewfolder/impl/file/ViewFileView.kt

```diff
@@ -195,25 +199,41 @@ private fun LineRow(
                 }
                 .padding(horizontal = 4.dp),
             text = line,
-            color = line.toColor(),
+            color = line.toColor(colorationMode),
             style = ElementTheme.typography.fontBodyMdRegular
         )
     }
 }
 
 /**
- * Convert a logcat line to a color.
- * Ex: `01-23 13:14:50.740 25818 25818 D org.matrix.rust.sdk: elementx: SyncIndicator = Hide | RustRoomListService.kt:81`
+ * Convert a line to a color.
+ * Ex for logcat:
+ * `01-23 13:14:50.740 25818 25818 D org.matrix.rust.sdk: elementx: SyncIndicator = Hide | RustRoomListService.kt:81`
+ *                                 ^ use this char to determine the color
+ * Ex for logs:
+ * `2024-01-26T10:22:26.947416Z  WARN elementx: Restore with non-empty map | MatrixClientsHolder.kt:68`
+ *                                  ^ use this char to determine the color, see [LogLevel]
```

Comment:
Use the last char since the level name is always 5 chars with end alignment. So we could not make the diff between ` INFO` and ` WARN`.